src/studio/schema/documents/person.ts

```diff
@@ -58,11 +58,11 @@ export default defineType({
       title: 'Role',
       type: 'string',
     }),
-    defineField({
-      name: 'biography',
-      title: 'Biography',
-      type: 'blockContent',
-    }),
+    // defineField({
```

Comment:
Why was this commented out?

Reply:
There is no reason, I'll reset the file to match trunk